app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -446,7 +446,11 @@ fun MapScreen(
                     modifier =
                         Modifier.fillMaxSize()
                             .testTag(MapScreen.TEST_TAG_MAP)
-                            .padding(bottom = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT))
+                            .padding(
+                                bottom =
+                                    MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT -
+                                        20.dp)) // -20.dp to avoid the map being to small under
```

Comment:
Nitpick: define this value in a constant in the `MapScreen` object. The constants should not be shared between screens, as we might want to make the bottom sheet slightly higher on details screen than on map screen for example.

Reply:
What do you mean? You mean add 20.dp as a constant in the MapScreen object?